python/fapolicy_analyzer/ui/ancillary_trust_database_admin.py

```diff
@@ -130,6 +130,7 @@ def on_deployBtn_clicked(self, *args):
         confirm_resp = confirmDialog.run()
         confirmDialog.hide()
         if confirm_resp == Gtk.ResponseType.YES:
+            self.system.deploy()
```

Comment:
Is there a return value or status check (or exception handling) associated with this call?